tests/HelpersTest.cpp

```diff
@@ -1309,7 +1309,7 @@ TEST_CASE("Types: TypeNameStrings", "[helpers]") {
     auto wsclass = CLI::detail::classify_object<std::wstring>::value;
     CHECK(CLI::detail::object_category::wstring_assignable == wsclass);
 
-#if defined CLI11_HAS_FILEYSTEM && CLI11_HAS_FILESYSTEM > 0 && defined(_MSC_VER)
+#if defined CLI11_HAS_FILESYSTEM && CLI11_HAS_FILESYSTEM > 0 && defined(_MSC_VER)
```

Comment:
Notably, this appears to be a broken guard

Reply:
hmmm

Reply:
this is probably triggering the coverage issue, meaning some bits of code were never run

Reply:
I'm happy to drop things and leave them for you to address separately.

Reply:
Yeah, the extra coverage in https://app.codecov.io/gh/CLIUtils/CLI11/pull/1101/blob/include/CLI/TypeTools.hpp definitely would make sense based on being able to run this additional line:
https://github.com/CLIUtils/CLI11/blob/bdd12bf308e90e911da0e58c83ce72179c3ce3ec/tests/HelpersTest.cpp#L1313

Reply:
I will see about getting the coverage back in a different PR